config/stylelint-config-carbon/index.js

```diff
@@ -7,17 +7,16 @@
 
 'use strict';
 
-module.exports = {
+export default {
   extends: [
-    require.resolve('stylelint-config-standard'),
-    require.resolve('stylelint-no-unsupported-browser-features'),
-    require.resolve('stylelint-config-idiomatic-order'),
-    require.resolve('stylelint-config-standard-scss'),
-    require.resolve('./rules/possible-errors'),
-    require.resolve('./rules/limit-language-features'),
-    require.resolve('./rules/stylistic-issues'),
-    require.resolve('./plugins/a11y'),
-    require.resolve('./plugins/prettier'),
```

Comment:
Majorly because of incompatibility between Stylelint 16 and `prettier: "^2.8.8"` and since Lee's PR also removed the ./plugins/prettier file. we planed bumping up prettier separately, it might make sense to deliver both updates in the same PR - Thoughts ? 

I tried moving to "prettier": "^3.2.5" and encountered - ERROR:  `prettier.resolveConfig.sync` is not a function.
Looks like we need to move to  "`eslint-plugin-prettier": "^5.0.0-alpha.1`" - [reference](https://github.com/prettier/eslint-plugin-prettier/issues/562)